src/item/FireworkRocket.php

```diff
@@ -40,23 +40,34 @@
 class FireworkRocket extends Item{
 
 	protected const TAG_FIREWORK_DATA = "Fireworks"; //TAG_Compound
-	protected const TAG_FLIGH_DURATION = "Flight"; //TAG_Byte
+	protected const TAG_FLIGHT_DURATION = "Flight"; //TAG_Byte
 	protected const TAG_EXPLOSIONS = "Explosions"; //TAG_List
 
 	protected int $flightDuration = 1;
 
 	/** @var FireworkRocketExplosion[] */
 	protected array $explosions = [];
 
+	/**
+	 * Returns the flight duration of the firework (equals the amount of gunpowder used in crafting the rocket).
+	 */
 	public function getFlightDuration() : int{
 		return $this->flightDuration;
 	}
 
-	public function setFlightDuration(int $duration) : void{
+	/**
+	 * Sets the flight duration of the firework.
```

Comment:
This is still confusing. It's not really the duration of flight at all. Rather it's a multiplier that is multiplied by a random value to get the actual flight duration.

Reply:
I have updated it, is it better for you?

Reply:
I think the functions should still be renamed. Perhaps "flight duration multiplier" would be a better name.